src/app/pages/ui/tables/tables.component.html

```diff
@@ -0,0 +1,302 @@
+            <th class="mdl-data-table__cell--non-numeric">Status</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr>
+            <td class="mdl-data-table__cell--non-numeric">54</td>
+            <td class="mdl-data-table__cell--non-numeric">The Hitchhiker's Guide to th...</td>
+            <td class="mdl-data-table__cell--non-numeric">Douglas Adams</td>
+            <td class="mdl-data-table__cell--non-numeric">1979</td>
+            <td class="mdl-data-table__cell--non-numeric"><span class="label label--mini color--orange">Unavailable</span> </td>
+          </tr>
+          <tr>
+            <td class="mdl-data-table__cell--non-numeric">55</td>
+            <td class="mdl-data-table__cell--non-numeric">Madame Bovary</td>
+            <td class="mdl-data-table__cell--non-numeric">Gustave Flaubert</td>
+            <td class="mdl-data-table__cell--non-numeric">1856</td>
+            <td class="mdl-data-table__cell--non-numeric"><span class="label label--mini background-color--primary">Available</span> </td>
+          </tr>
+          <tr>
+            <td class="mdl-data-table__cell--non-numeric">56</td>
+            <td class="mdl-data-table__cell--non-numeric">David Copperfield</td>
+            <td class="mdl-data-table__cell--non-numeric">Charles Dickens</td>
+            <td class="mdl-data-table__cell--non-numeric">1850</td>
+            <td class="mdl-data-table__cell--non-numeric"><span class="label label--mini background-color--primary">Available</span> </td>
+          </tr>
+          <tr>
+            <td class="mdl-data-table__cell--non-numeric">57</td>
+            <td class="mdl-data-table__cell--non-numeric">War and Peace</td>
+            <td class="mdl-data-table__cell--non-numeric">Leo Tolstoy</td>
+            <td class="mdl-data-table__cell--non-numeric">1869</td>
+            <td class="mdl-data-table__cell--non-numeric"><span class="label label--mini background-color--primary">Available</span> </td>
+          </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <div class="mdl-cell mdl-cell--6-col-desktop mdl-cell--6-col-tablet mdl-cell--4-col-phone">
+    <div class="mdl-card">
+      <div class="mdl-card__title">
+        <h1 class="mdl-card__title-text">Borderless table</h1>
+      </div>
+      <div class="mdl-card__supporting-text no-padding">
+        <table class="mdl-data-table mdl-js-data-table borderless-table">
+          <thead>
+          <tr>
+            <th class="mdl-data-table__cell--non-numeric">#</th>
+            <th class="mdl-data-table__cell--non-numeric">Book title</th>
+            <th class="mdl-data-table__cell--non-numeric">Author</th>
+            <th class="mdl-data-table__cell--non-numeric">First published</th>
+            <th class="mdl-data-table__cell--non-numeric">Status</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr>
+            <td class="mdl-data-table__cell--non-numeric">31</td>
```

Comment:
it's better to take out all values to array constant and use loops for table rows